v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,306 @@
+      -out node.csr \
+      -batch
+    ~~~
+
+4. Sign the node CSR to create the node certificate for the first node using the [`openssl ca`](https://wiki.openssl.org/index.php/Manual:Ca(1)) command. 
+
+    We recommend that you use the CockroachDB default value of the node certificate expiration period, which is 1830 days. You can set the CA certificate expiration period using the `days` flag.
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Sign the CSR using the CA key.
+    $ openssl ca \
+      -config ca.cnf \
+      -keyfile my-safe-directory/ca.key \
+      -cert certs/ca.crt \
+      -policy signing_policy \
+      -extensions signing_node_req \
+      -out certs/node.crt \
+      -outdir certs/ \
+      -in node.csr \
+      -days 1830 \
+      -batch
+    ~~~
+
+5. Upload certificates to the first node:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Create the certs directory:
+    $ ssh <username>@<node1 address> "mkdir certs"
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Upload the CA certificate and node certificate and key:
+    $ scp certs/ca.crt \
+    certs/node.crt \
+    certs/node.key \
+    <username>@<node1 address>:~/certs
+    ~~~
+
+6. Delete the local copy of the first node's certificate and key:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ rm certs/node.crt certs/node.key
+    ~~~
+
+    {{site.data.alerts.callout_info}}This is necessary because the certificates and keys for additional nodes will also be named <code>node.crt</code> and <code>node.key</code>.{{site.data.alerts.end}}
+
+7. Repeat steps 1 - 6 for each additional node.
+
+### Create the certificate and key pair for a client
+
+1. Create the client.cnf file for the first client:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # OpenSSL client configuration file
+    [ req ]
+    prompt=no
+    distinguished_name = distinguished_name
+
+    [ distinguished_name ]
+    organizationName = Cockroach
+    commonName = <username>
+    ~~~
+
+    {{site.data.alerts.callout_info}}The <code>commonName</code> parameter is vital for CockroachDB functions. You can modify or leave out other parameters as per your preferred OpenSSL configuration, but do not leave out the <code>commonName</code> parameter.  {{site.data.alerts.end}}
+  
+
+2. Create the key for the first client using the [`openssl genrsa`](https://wiki.openssl.org/index.php/Manual:Genrsa(1)) command:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ openssl genrsa -out certs/client.<username>.key 2048 
+    $ chmod 400 certs/client.<username>.key
+    ~~~
+
+3. Create the CSR for the first client using the [`openssl req`](https://wiki.openssl.org/index.php/Manual:Req(1)) command:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Create client certificate signing request
+    $ openssl req \
+      -new \
+      -config client.cnf \
+      -key certs/client.maxroach.key \
```

Comment:
we've been using `<username>` in the config file and commands above. We should do the same here, or use `maxroach` everywhere.